### PR TITLE
Add stale PR policy

### PR DESCRIPTION
**Pull request by viglesiasce**
First cut at a stale PR policy. Happy to make changes and alter as necessary.

### CODE REVIEW

**Review by prydonius — APPROVED** [review]
lgtm!

**Comment by lachie83** [comment]
lgtm

**Comment by unguiculus** [comment]
lgtm

**Comment by yuvipanda** [comment]
Should this be merged before it becomes stale? :D

File: README.md [hunk 1]
```diff
@@ -61,6 +61,9 @@ The following outlines the review procedure used by the Chart repository maintai
 * ***UX REVIEWED*** - The chart installation UX has been reviewed and found to be satisfactory. (may happen in parallel to CODE REVIEWED)
 * ***LGTM*** - Added ONLY once both UX/CODE reviewed are both present. Merge must be handled by someone OTHER than the maintainer that added the LGTM label. This label indicates that given a quick pass of the comments this change is ready to merge
 
+### Stale Pull Requests
+
+After initial review feedback, if no updates have been to the pull request for 1 week, the `stale` label will be added. If after another week there are still no updates it will be closed. Please re-open if/when you have made the proper adjustments.
```
Comment (unguiculus) [inline README.md]: > if no updates have been to the pull request

I'm not a native speaker, but this doesn't quite sound like correct English to me. Shouldn't it rather be `if no updates have been made to the pull request`?

Comment (prydonius) [reply README.md]: good catch :+1:, apparently your English is better than ours since we missed this ;)

**Pull request merged**